src/main/scala/firrtl/passes/memlib/MemUtils.scala

```diff
@@ -46,6 +46,7 @@ object createMask {
   def apply(dt: Type): Type = dt match {
     case t: VectorType => VectorType(apply(t.tpe), t.size)
     case t: BundleType => BundleType(t.fields map (f => f copy (tpe=apply(f.tpe))))
+    case GroundType(w) if w == IntWidth(0) => UIntType(IntWidth(0))
```

Comment:
Is there a nicer syntax for this?

Reply:
How about
```case GroundType(IntWidth(0)) => UIntType(IntWidth(0))```
?

Reply:
You'd think so, but the unapply for intwidth wants a BigInt, and `GroundType(IntWidth(BigInt(0)))` gave me some other weird error that I didn't really understand (maybe `BigInt` has its own unapply doing something weird?).

Reply:
Yeah, it doesn't work. What you can do is somewhere define:
```val zeroW = IntWidth(0)```
and then you can do:
```case GroundType(`zeroW`) => ...```

Reply:
Ah, good point- I rarely use the backticks, but this is a good example where it would be good. I think it's a wash and I'll just leave the existing syntax. Thanks!